assignments/objects.js

```diff
@@ -19,26 +19,83 @@ const example = {
 
 // Write your intern objects here:
 
+const hireOne = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy@psu.edu",
+  "gender": "F",
+  "speak": function(){
+    return "Hello, my name is " + this.name + "!";
+  }
+}
+
+const hireTwo = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": function(){
+    return "Hello, my name is " + this.name + "!";
+  }
+}
+
+const hireThree = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M",
+  "speak": function(){
+    return "Hello, my name is " + this.name + "!";
+  }
+}
+
+const hireFour = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M",
+  "speak": function(){
+    return "Hello, my name is " + this.name + "!";
+  }
+}
+
+const hireFive = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  "speak": function(){
+    return "Hello, my name is " + this.name + "!";
+  },
+  "multiplyNums": function(num1, num2){
+    return num1 * num2;
+  }
+}
```

Comment:
Kudos for putting your interns' associated action functions in their actual objects.